src/mock/nethserver-7-x86_64.cfg

```diff
@@ -13,6 +13,7 @@ config_opts['plugin_conf']['ccache_enable'] = False
 config_opts['macros']['%url_prefix'] = 'http://github.com/NethServer'
 config_opts['macros']['%vendor'] = 'NethServer'
 config_opts['macros']['%dist'] = 'ns7'
+config_opts['rpmbuild_networking'] = True
```

Comment:
This parameter must be set in sites-default.cfg

Reply:
Are you sure? Is this needed also for ns 6 builds? It should't because ns 6 build doesn't use nspawn
```
# If you're using nspawn, then by default networking will be turned off
# for rpmbuild.  This helps ensure more reproducible builds.
# config_opts['rpmbuild_networking'] = False
```